app/views/staff/shared/xml/_transaction.xml.haml

```diff
@@ -0,0 +1,17 @@
+%transaction{"ref" => transaction.reference}
+  %transaction-type{"code" => transaction.transaction_type}
+  %transaction-date{"iso-date" => transaction.date}
+  %value{"currency" => transaction.currency, "value-date" => transaction.date}=transaction.value
+  %description
+    %narrative=transaction.description
+  %provider-org{"provider-activity-id" => "", "type" => transaction.provider.organisation_type, "ref" => ""}
+    %narrative=transaction.provider.name
+  %receiver-org{"receiver-activity-id" => "", "type" => transaction.receiver.organisation_type, "ref" => ""}
+    %narrative=transaction.receiver.name
+  %disbursement-channel{"code" => transaction.disbursement_channel}
```

Comment:
I don't believe we need to show these fields at the transaction level too, given we aren't allowing the user to override the defaults set by the parent activity.

On the Trello card we have the following that aren't needed within the transaction XML:

```
disbursement-channel
recipient-country
recipient-region
flow-type
finance-type
aid-type
tied-status
```